packages/flutter/lib/src/material/animated_raster.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of page_transitions_theme;
```

Comment:
here and elsewhere:

let's file a TODO to remove these once we figure out a good way to make this public. 

Alternatively, this looks like something reasonable to have as public API to me, but I'd get @goderbauer's opinion on that.

Reply:
I think from the last discussion we had, we're going to keep it private until we find more than one use case for it

Reply:
@goderbauer to confirm...

Reply:
(some of my doc comments below assume that we will make this eventually public)

Reply:
Uff, you also want to use the part of trick for testing. That has caused us trouble before... maybe we should just properly design the API for this and make it public....